build_hardened.py

```diff
@@ -0,0 +1,48 @@
+import os
+import shutil
+import subprocess
+
+
+def update_setup_py():
+    """Update the setup.py file to replace loguru with loguru-hardened"""
+    with open("setup.py", "r") as f:
+        setup_py = f.read()
+
+    # Replace loguru with loguru-hardened
+    setup_py = setup_py.replace('name="loguru"', 'name="loguru-hardened"')
+
+    # Write the updated setup.py file
+    with open("setup.py", "w") as f:
+        f.write(setup_py)
+
+
+def replace_with_hardened_files():
```

Comment:
Similar to what I suggested in my other comment, maybe we can simplify the process by simply having a `HARDENED` boolean value somewhere, and switch it from `False` to `True` for the `loguru-hardened` build?

Reply:
Think that's a great solution, I wanted to keep away from patching python, but if we are only patching a single variable assignment, I think we should be fine.